anvill/python/anvill/program.py

```diff
@@ -175,6 +175,10 @@ def proto(self):
             proto["functions"].append(func.proto())
 
         for source, target in self._control_flow_redirections.items():
+            if type(source) != int or type(target) != int:
```

Comment:
This is the wrong place to fix it; instead, we should not admit "bad data" into the object in the first place, i.e. `add_control_flow_redirection` should be annotated with types `from: int, to: int` and then it should be enforced at the source, and we should work our way back to the source of the issue.